gofsutil_fs_test.go

```diff
@@ -0,0 +1,1566 @@
+//go:build linux || darwin
+// +build linux darwin
+
+// Copyright © 2022-2024 Dell Inc. or its subsidiaries. All Rights Reserved.
```

Comment:
Please ensure the year is updated in all the files, if it hasn't been done already.
```suggestion
// Copyright © 2025 Dell Inc. or its subsidiaries. All Rights Reserved.
```

Reply:
Done